ai_coach/ai_coach.py

```diff
@@ -198,8 +198,18 @@ def ask_from_coach(self, data, suffix=''):
         if self.feedback_count >= self.feedback_threshold:
             return {'error': _("You've exhausted all available chances to ask the coach for help")}
 
+        ai_context = self.context + """
+        Evaluate the student's response to the question below:
```

Comment:
I think we should make it first person rather than 3rd person.

```suggestion
        Evaluate my answer to the question below:
```

Otherwise, the AI would give the answer in 3rd person format as well.

Reply:
Yes, I will fix this. Thanks for pointing out.